tests/test_config.py

```diff
@@ -611,19 +611,33 @@ def test_specifiers(tmp_path: Path) -> None:
         Architecture=arm64
 
         [Output]
+        ImageId=my-image-id
+        ImageVersion=1.2.3
         OutputDirectory=abcde
         Output=test
 
-        [Distribution]
-        ImageId=%d~%r~%a
-
-        [Output]
-        CacheDirectory=%O/%o
+        [Content]
+        Environment=DISTRIBUTION=%d
+                    RELEASE=%r
+                    ARCHITECTURE=%a
+                    IMAGE_ID=%i
+                    IMAGE_VERSION=%v
```

Comment:
That doesn't test all that much, since we set these environment variables to the values in `load_environment` anyway, no?

Reply:
oops didn't notice that, let me set make the pascal case